### PR TITLE
[MM-58517] Migrate tooltips of "components/rhs_card_header/rhs_card_header.tsx" to WithTooltip

### LINKED ISSUE #27229: Migrate tooltips of "components/rhs_card_header/rhs_card_header.tsx" to WithTooltip
The purpose of this ticket is to eliminate the usage of OverlayTrigger from “components/overlay_trigger” and Tooltip of “components/tooltip” in the file `webapp/channels/src/components/rhs_card_header/rhs_card_header.tsx`. These should be replaced with the WithTooltip component from “components/with_tooltip”.
For reference, please follow these PRs: 
- https://github.com/mattermost/mattermost/pull/26458
- https://github.com/mattermost/mattermost/pull/27146
- https://github.com/mattermost/mattermost/pull/27145

----
If you're interested please comment here and come [join our "Contributors" community channel](https://community.mattermost.com/core/channels/tickets) on our daily build server, where you can discuss questions with community members and the Mattermost core team. For technical advice or questions, please  [join our "Developers" community channel](https://community.mattermost.com/core/channels/developers).

New contributors please see our [Developer's Guide](https://developers.mattermost.com/contribute/getting-started/).

JIRA: https://mattermost.atlassian.net/browse/MM-58517

### PR DESCRIPTION
**Pull request by Dev-Arhaan**
#### Ticket Link
Fixes https://github.com/mattermost/mattermost/issues/27229
https://mattermost.atlassian.net/browse/MM-58517

#### Release Note

```release-note

```

### CODE REVIEW

File: webapp/channels/src/components/rhs_card_header/rhs_card_header.tsx [hunk 1]
```diff
@@ -61,47 +61,47 @@ class RhsCardHeader extends React.PureComponent<Props> {
...
                     />
-                </Tooltip>
+                </WithTooltip>
             );
             break;
         }
 
         const closeSidebarTooltip = (
-            <Tooltip id='closeSidebarTooltip'>
+            <WithTooltip id='closeSidebarTooltip'>
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/rhs_card_header/rhs_card_header.tsx:95]: We dont have to replace Tooltip component with WithTooltip but only for OverlayTrigger we need to do that for this one
```suggestion
            
```
```
const closeSidebarTooltip = <FormattedMessage
                    id='rhs_header.closeSidebarTooltip'
                    defaultMessage='Close'
                />
```

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]
Can you please check the failing CIs

File: webapp/channels/src/components/rhs_card_header/rhs_card_header.tsx (hunk 1 above)
Comment (Dev-Arhaan) [reply webapp/channels/src/components/rhs_card_header/rhs_card_header.tsx:95]: ok Let me try to push that by tomorrow

**Comment by M-ZubairAhmed** [comment]
@Dev-Arhaan do you plan to reopen this PR for future work?

**Comment by hmhealey** [comment]
This was replaced by https://github.com/mattermost/mattermost/pull/27501

**Pull request closed without merging**